src/app/profiles/(components)/MultipleChoiceComponent.tsx

```diff
@@ -0,0 +1,162 @@
+'use client'
+
+import {Button} from "@/components/ui/button"
+import {
+    Card,
+    CardContent,
+    CardDescription,
+    CardFooter,
+    CardHeader,
+    CardTitle,
+} from "@/components/ui/card"
+import { Label } from "@/components/ui/label"
+import { RadioGroup, RadioGroupItem } from "@/components/ui/radio-group"
+import { SortableContext, verticalListSortingStrategy, useSortable, arrayMove } from "@dnd-kit/sortable"
+import { DndContext, DragEndEvent, closestCenter } from '@dnd-kit/core'
+import { CSS } from '@dnd-kit/utilities'
+import { Formik, Form, useFormikContext } from 'formik'
+import Trash from '../../../../public/trash-icon.svg'
+import Drag from '../../../../public/drag-icon.svg'
+import { Input } from "@/components/ui/input"
+
+type Answer = {
+    id: number
+    label: string
+}
+
+const INITIAL_QUESTION: string = "What is your favorite color?"
+const INITIAL_ANSWERS: Answer[] = [
+    {id: 1, label: ""},
+]
+const INITIAL_VALUES = {
```

Comment:
kinda-nit;

these would most likely be passed in to `<MultipleChoiceComponent />` from some upper component that retrieved them from the db.

I would just define our fake initial values directly within `<MultipleChoiceComponent />` for now